bin/test_inasafe.py

```diff
@@ -30,64 +29,67 @@ class TestInasafeCommandLine(unittest.TestCase):
     def setUp(self):
         self.args = CommandLineArguments({
             '--download': False,
-            '--exposure': 'safe/test/data/exposure/buildings.shp',
-            '--extent': '106,8525945:-6,2085970:106,7999364:-6,1676174',
-            '--hazard': 'safe/test/data/hazard/continuous_flood_20_20.asc',
-            '--impact-function': 'FloodRasterBuildingFunction',
+            '--hazard':
+                '../safe/test/data/gisv4/hazard/tsunami_vector.geojson',
+            '--exposure':
+                '../safe/test/data/gisv4/exposure/raster/population.asc',
+            '--extent': '106,7999364:-6,2085970:106,8525945:-6,1676174',
+            '--aggregation': None,
```

Comment:
We should explain how this will interact with the extent option (supercedes extents? filtered by extent? clipped by extent?)

Reply:
It's not possible to run IF with both extent and aggregation set, so does the CLI now..